### PR TITLE
Merging project three code

### PR DESCRIPTION
**Pull request by kianaomoerah**
- created Cart, Inventory component for structure of app 
- connected firebase database of products to display on page 
- tracked state appropriately amongst components to render products from database, and allow the update of cart

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,59 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+      <>
+        <Header />
```
Comment (CourtG) [inline src/App.js:51]: chefs kiss... i love how tight this code is!

**Review by CourtG — COMMENTED** [review]
I know, I literally left one comment, and it's a compliment lol. I really can't see anything wrong with your code. It's so well organized and you have what feels like the perfect amount of components! 

I see on the live site that the prices seem to be moving around on you but I know that's just some CSS tweeking. This looks like it was a lot of work!! firebase AND an API, well done!!

File: src/App.css [hunk 2]
```diff
@@ -1,38 +1,192 @@
...
+  width: 75%;
+  margin: 0 auto;
+  padding: 25px;
+  align-self: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+h2 {
```
Comment (LaurynFerkul) [inline src/App.css]: Great styling! You could probably just move this color to the other h2 selector

**Review by LaurynFerkul — COMMENTED** [review]
Great job! Your code looks so clean!

**Pull request merged**